### DISCUSSION TITLE
Workaround for missing VerticalContentAlignment property on TextBlock

### DISCUSSION THREAD

**Discussion by LaurentInSeattle** [AvaloniaUI/Avalonia#15764] [Q&A] [answered]
Hi all, 

The Avalonia TextBlock has no VerticalContentAlignment property.
Is there a (preferably simple) workaround for centering the text vertically ? 

Thanks!

**Comment by timunie** [comment, marked as answer]
Why not center the TextBlock instead? You can wrap it in a Panel if you want to have more control over the exact position

**Reply by LaurentInSeattle** [reply]
Thanks, also works with wrapping it within a border
